pkgs/applications/editors/emacs/elisp-packages/libgenerated.nix

```diff
@@ -45,6 +45,18 @@ let
           inherit sha256 url;
         }
       ) {};
+      sourcehut = self.callPackage ({ fetchzip }:
```

Comment:
Was this script-generated?

Reply:
Also, is it compatible with the generator scripts `update-*`?

Reply:
> Was this script-generated?
No, i added it manually.

> Also, is it compatible with the generator scripts update-*?
compatible how? the update-* scripts don't touch libgenerated.nix?